### PR TITLE
Update team names

### PR DESCRIPTION
**Pull request by jonathanglassman**
Replace out of date team names:

- Data Products (formerly Data Labs)
- Data Infrastructure (formerly Data AAA)
- Data Insights (formerly [GOV.UK](http://gov.uk/) Insights) (edited)

Update email addresses and change GOV.UK data community to Data Services.

### CODE REVIEW

File: source/index.html.md.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@ review_in: 6 months
 
 # GOV.UK data community technical documentation
 
-The GOV.UK data community is made up of several teams including GOV.UK Insights and Data Labs.
+The GOV.UK data community is made up of several teams including Data Insights and Data Products.
```
Comment (mammykins) [inline source/index.html.md.erb:10]: The GOV.UK data community could be replaced with GDS Data Services. I think that is our new preferred name, confirm in our slack convo with other knowlegeable parties

File: source/analysis/intent-detector/iunis/page-term-idf-matrix/index.html.md.erb [hunk 2]
```diff
@@ -28,7 +28,7 @@ You can now [go to the Jupyter notebooks directory](#go-to-the-jupyter-notebooks
 
 1. Open the [`govuk-intent-detector-page-term-tf-idf` AWS instance](https://eu-west-1.console.aws.amazon.com/sagemaker/home?region=eu-west-1#/notebook-instances/govuk-intent-detector-page-term-tf-idf).
 
-    You must have the `govuk-datascienceusers` AWS IAM role to access this instance. [Contact the GOV.UK Data Labs team](mailto:gov.uk-data-labs@digital.cabinet-office.gov.uk) for more information.
+    You must have the `govuk-datascienceusers` AWS IAM role to access this instance. [Contact the GOV.UK Data Products team](mailto:gov.uk-data-labs@digital.cabinet-office.gov.uk) for more information.
```
[the remark this answers is not in the document]
Comment (jonathanglassman) [reply source/analysis/intent-detector/iunis/page-term-idf-matrix/index.html.md.erb:31]: It has - data-products@digital.cabinet-office.gov.uk - i'll replace all instances

File: source/index.html.md.erb (hunk 1 above)
Comment (jonathanglassman) [reply source/index.html.md.erb:10]: have replaced

**Comment by jonathanglassman** [comment]
> Team names have been updated. There is a question around some of the respective email addresses associated with them that may also need updating. To check.
> 
> The data group itself (i.e. the group that is composed of these teams) has also been renamed. This name should be confirmed and corrected in this PR too.

Have actioned @mammykins

**Review by mammykins — APPROVED** [review]
Great, thanks! :shipit:

**Pull request merged**